### DISCUSSION TITLE
【灌水】跟大家讲个有趣的事情

### DISCUSSION THREAD

**Discussion by Robin-txdy** [share121/inter-knot#4814] [灌水]
![IMG_20231020_160625](https://github.com/user-attachments/assets/33555096-be43-483b-b893-a4f758d6ed20)
我跟我朋友在玩你画我猜的时候，本来有一个答案是［嫁出去的女儿，泼出去的水］，我当时嘴比脑子快，说成了［泼出去的女儿，嫁出去的水］，我跟我朋友笑了好久

**Comment by love262-wkx** [comment]
这真的很nb

**Comment by lllliufeng** [comment]
好家伙<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by meillml** [comment]
6

**Comment by Junjun280** [comment]
绷不住了，哈哈哈

**Comment by Dave-12138** [comment]
一定是枫丹人吧（
![](https://s2.loli.net/2024/08/31/4raExbCUDPh861n.webp)
![](https://s2.loli.net/2024/08/31/Tzr2mjb17WvniDL.webp)

**Comment by 070612wwz** [comment]
我嘞个六啊
